frigate/http.py

```diff
@@ -251,6 +246,24 @@ def set_sub_label(id):
     )
 
 
+@bp.route("/sub_labels")
+def get_sub_labels():
+    try:
+        events = (
+            Event.select(
```

Comment:
I think this should be `Event.select(Event.sub_label).distinct()`.

Reply:
Yeah that makes more sense